spec/models/recipe_spec.rb

```diff
@@ -12,4 +26,12 @@
     it {should have_many :recipe_ingredients}
     it {should have_many(:ingredients).through(:recipe_ingredients)}
   end
+
+  describe "instance methods" do
+    describe "#total_cost" do
+      it "sums the total cost of all ingredients for a specific recipe" do
+        expect(@ez_ziti.total_cost).to eq(19)
+      end
+    end
+  end
```

Comment:
```ruby
def total_cost
  19
end
```

Does your assertion disprove that this is what your method looks like? Probably not. My suggestion, for instance methods, is to test more than 1 instance object so that way your code proves that the value isn't hard-coded and is dynamic.

```ruby
expect(@ez_ziti.total_cost).to eq(19)
expect(@spaghetti.total_cost).to eq(12) # create another recipe
```